apps/voting/app/src/screens/VoteDetail.js

```diff
@@ -50,6 +52,10 @@ function VoteDetail({ vote, onBack, onVote, onExecute }) {
     numData,
     voteId,
   } = vote
+
+  // TODO: get real disputable vote info. this is temporary
+  vote.disputable = getDisputableVoteById(vote.voteId)
```

Comment:
It's a TODO so ignore if it's not relevant but creating a new object rather than mutating would be preferred:

```const disputableVote = {...vote, disputable: getDisputableVoteById(vote.voteId)}```

Reply:
This is temporary. This function wont exist and the info of vote.disputable will come with the vote all along from the script. So it wont mutate. Is just a temporary fix so I can show diferent mock data on each vote.